config/demo

```diff
@@ -6,7 +6,7 @@ export api_url = http://api.demov6.camptocamp.org
 export api_url_internal = http://api
 export api_url_host = api
 
-export discourse_url = https://forum.demov6.camptocamp.org
+export discourse_url = http://forum.demov6.camptocamp.org
```

Comment:
Why do you need this? Perhaps you have forgotten to undo this change?

Reply:
indeed it's a forget, all the change in this file are an error.

Reply:
OK, please undo that change as well and I think it's OK.